src/selectors/WalletSelectors.js

```diff
@@ -16,32 +16,16 @@ export function getSelectedCurrencyWallet(state: RootState): EdgeCurrencyWallet
   return state.core.account.currencyWallets[state.ui.wallets.selectedWalletId]
 }
 
-export const getActiveWalletCurrencyCodes = (state: RootState) => {
-  const { account } = state.core
-  const { activeWalletIds, currencyWallets } = account
-  const currencyCodesMap = activeWalletIds.reduce((map, id) => {
-    const wallet = currencyWallets[id]
-    if (!wallet) return map
+export const getActiveWalletCurrencyInfos = (currencyWallets: { [walletId: string]: EdgeCurrencyWallet }): EdgeCurrencyInfo[] => {
```

Comment:
This `getActiveWalletCurrencyInfos` is used in the previous commit, but doesn't get defined until this commit. You could just squash this commit with the previous one to fix it, or just not worry about it.